src/aleph/vm/conf.py

```diff
@@ -224,22 +226,16 @@ class Settings(BaseSettings):
         description="Address of the account receiving payments",
     )
     # This address is the ALEPH SuperToken on SuperFluid Testnet
-    PAYMENT_SUPER_TOKEN: str = Field(
-        default="0xc0Fbc4967259786C743361a5885ef49380473dCF",  # Mainnet
-        # default="0x1290248e01ed2f9f863a9752a8aad396ef3a1b00",  # Testnet
-        description="Address of the ALEPH SuperToken on SuperFluid",
-    )
     PAYMENT_PRICING_AGGREGATE: str = ""  # TODO: Missing
 
-    PAYMENT_RPC_API: HttpUrl = Field(
-        default="https://api.avax.network/ext/bc/C/rpc",
-        # default="https://api.avax-test.network/ext/bc/C/rpc",
-        description="Default to Avalanche Testnet RPC",
+    RPC_AVAX: HttpUrl = Field(
+        default=None,
+        description="Replace chain RPC for AVAX",
     )
-    PAYMENT_CHAIN_ID: int = Field(
-        default=43114,  # Avalanche Mainnet
-        # default=43113,  # Avalanche Fuji Testnet
-        description="Avalanche chain ID",
+
+    RPC_BASE: HttpUrl = Field(
+        default=None,
+        description="Replace chain RPC for BASE",
```

Comment:
replace?

Reply:
If RPC_BASE is not None, we will replace the value here in [Chain.py](https://github.com/aleph-im/aleph-vm/blob/4aabb55cca2e24d2f781ea69ac761faee286a6d1/src/aleph/vm/orchestrator/chain.py#L48).

The data structure we were using was not the simplest to edit via Pydantic settings.

It would also make sense in the future to directly use aleph-sdk-python. (and would make additions of new EVM chain even more simple)

Reply:
ok I took the liberty of directly modifying the description and symplying the setting logic